app/javascript/utilities/getFocusTrapToggle.js

```diff
@@ -0,0 +1,32 @@
+import { createFocusTrap } from 'focus-trap';
+
+export function getFocusTrapToggle(containerSelector) {
```

Comment:
This helper wraps up the `focus-trap` code along with the previous `hidden` toggle functionality used in the non-Preact modals we have